modules/custom-access-flow/strategy.py

```diff
@@ -63,8 +72,17 @@ def deescalate(self, target_id, event):
         # Here is where you might hit an external API to revoke access from the requester
         # response = requests.post(
         #     "https://some-saas-product.com/api/v1/revoke-access",
+        #     headers={"Authorization": self.get_api_token()},
         #     json={"email": requester}
         # )
 
         # if not response.ok:
         #     raise RuntimeError(f"Failed to revoke access: {response.json()['error']}")
+
+    def get_api_token(self):
+        secrets = self.integration.settings["secrets"]
```

Comment:
since this is an example, would it be helpful to put type annotations so that people know where to look for type hints?

```suggestion
        secrets: List[Secret]= self.integration.settings["secrets"]
```